### ISSUE TITLE
Does not support normal HTML

### ISSUE THREAD

**Issue by Andrewyg** [evilstreak/markdown-js#295]
Normally markdown editor should support html tag but if you write some thing like `<a>hi</a>` it'll become something like this `<p>&lt;a&gt;hi&lt;/a&gt;</p` this should be fine but how about `<h1>hi</h1>` it'll become `<p><h1>hi</h1></p>` where the browser will show **<h1>hi</h1>**

**Comment by begogna** [comment]
I have found the same problem !

**Comment by jduprey** [comment]
Same here.  markdown-js should honor existing HTML markup w/in the text, instead, it escapes it.  Please fix?

**Comment by 3stacks** [comment]
Any update?

**Comment by jduprey** [comment]
marked - seems to be an acceptable replacement.
https://github.com/chjj/marked

**Comment by 3stacks** [comment]
@jduprey, I just switched over to this library last night. It's working perfectly for me, as you can override each individual render method, which allowed me to do all intermediate manipulations I needed.

Can't recommend it enough.

**Comment by jduprey** [comment]
Yes. I, too, switched slightly after my original post and it seems to work great.
